test/distributions/test_distributions.py

```diff
@@ -1421,7 +1421,7 @@ def ref_log_prob(ref_rate, idx, x, log_prob):
         # theoretical results.
         dist = Poisson(rate_zero)
         dist.log_prob(torch.ones_like(rate_zero)).backward()
-        torch.testing.assert_allclose(rate_zero.grad, torch.inf)
+        self.assertEqual(rate_zero.grad, torch.inf)
```

Comment:
Comparing scalar tensors to Python scalars is not supported by `torch.testing.assert_close`. I've opted to use `self.assertEqual` here since that still includes this type wrangling for BC.